site/content/en/docs/tasks/run/statefulset.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Run StatefulSet"
+linkTitle: "StatefulSet"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a StatefulSet as a Kueue-managed workload.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running StatefulSets.
+
+We demonstrate how to support scheduling StatefulSets in Kueue based on the Plain Pod integration with the help of PodGroup,
+where StatefulSet is represented as a single independent workload.
+
+This guide is for [serving users](/docs/tasks#serving-user) that have a basic understanding of Kueue.
+For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+
+2. Follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
+   to learn how to enable the `v1/pod` integration and how to configure it using the `podOptions` field.
+   Also, ensure that you have the `v1/statefulset` integration enabled.
+
+   A configuration for Kueue with enabled statefulset integration would look like follows:
+   ```yaml
+   apiVersion: config.kueue.x-k8s.io/v1beta1
+   kind: Configuration
+   integrations:
+     frameworks:
+      - "pod" # required by statefulset
+      - "statefulset"
+     podOptions:
+       # You can change namespaceSelector to define in which 
+       # namespaces kueue will manage the pods.
+       namespaceSelector:
+         matchExpressions:
+         - key: kubernetes.io/metadata.name
+           operator: NotIn
+           values: [ kube-system, kueue-system ]
+       # Kueue uses podSelector to manage pods with particular 
+       # labels. The default podSelector will match all the pods. 
+       podSelector:
+         matchExpressions:
+         - key: kueue-job
+           operator: In
+           values: [ "true", "True", "yes" ]
```

Comment:
I think this is not necessary here as this is repeating the content on the Plain pod page. 

Or if you want we could make it minimal: 

2. Ensure that you have the `v1/statefulset` integration enabled, for example:
```yaml
   apiVersion: config.kueue.x-k8s.io/v1beta1
   kind: Configuration
   integrations:
     frameworks:
      - "pod" # required by statefulset
      - "statefulset"
```
Also, follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
   to learn how to enable the `v1/pod` integration and how to configure it using the `podOptions` field.

Reply:
I would prefer second one it's more understandable I think.